exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var percStudents = Math.round((15/23)*100); 
```

Comment:
It would be more understandable if you created a "total" variable (as you did in exercise G and did "numberOfStudents/total".

Reply:
Done